package.json

```diff
@@ -2,7 +2,7 @@
   "name": "print-js",
   "homepage": "http://printjs.crabbly.com",
   "description": "A tiny javascript library to help printing from the web.",
-  "version": "1.6.0",
+  "version": "1.6.1",
```

Comment:
I bumped the version but we can't use it on web project for github forked packages